sharedgo/deletionPicker.go

```diff
@@ -0,0 +1,21 @@
+// Picks what file from 10 to 50 (files 0-9 kept in the system for persistence) to delete
```

Comment:
Add a comment with the LICENSE header and author line.

Reply:
done!